doc/intl.md

```diff
@@ -0,0 +1,192 @@
+
+### Android 7.0 - 7.1 (SDK<26) and older
+
+- getCanonicalLocales: Unicode/CLDR version differences Expected SameValue(«und-u-ms-imperial», «und-u-ms-uksystem») to be true
+
+### Android 7.0 - 7.1 (SDK<24) and older
+
+- Collator: Doesn&#39;t canonically decompose the input strings. Canonically equivalent string with non-identical code points may not match.
+- getCanonicalLocales: Unicode/CLDR version differences (Expected SameValue(«und-u-ca-ethiopic-amete-alem», «und-u-ca-ethioaa») to be true; Expected SameValue(«und-u-ks-primary», «und-u-ks-level1») to be true)
+- Unit style does not work.
+- There are issues in the precision configuration due to lack of APIs.
+- DateFormat: There are issues with the calendar configuration which needs to be dug into.
+
+### SDK<21 and older
+
+On platforms before 21, Locale.forLanguageTag() is not available, hence we can&#39;t construct java.util.Locale object from locale tag. Hence, we fallback to English for any locale input.
+
+# Internationalization framework in Android Platform
+
+Our implementation is essentially projecting the Android platform provided internationalization facilities through the ECMA-402 specified services. It implies that the results of running the same code may vary between devices running different versions of Android.
+
+Android platform internationalization libraries have been based on [ICU4j project](https://unicode-org.github.io/icu-docs/#/icu4j). Version of ICU4j and the backing [CLDR data](http://cldr.unicode.org/) varies across Android platform versions. Also, the ICU APIs were never exposed directly, but only through wrappers and aliases. This results in significant variance in internationalization API surface and data across platform versions.
+
+The following table summarizes ICU, CLDR and Unicode versions available on the Android platforms that we intent to support.
+
+**Platform 24+ where ICU4j APIs are available.**
+| Android Platform Version | ICU | Unicode | CLDR
+| --- | --- | --- | --- |
+| Android 11 (API level 30) | ICU4J 66.1 ([ref](https://android.googlesource.com/platform/external/icu/+/refs/heads/android11-mainline-release/icu4j/readme.html)) | Unicode 13 beta | CLDR 36.1 |
+| Android 10 (API level 29) | ICU4j 63.2 ([ref](https://developer.android.com/guide/topics/resources/internationalization)) | CLDR 34 | Unicode 11.0 |
+| Android 9 (API level 28) | ICU4j 60.2 ([ref](https://developer.android.com/guide/topics/resources/internationalization)) | CLDR 32.0.1 | Unicode 10.0 |
+| Android 8.0 - 8.1 (API levels 26 - 27) | ICU4j 58.2( [ref](https://developer.android.com/guide/topics/resources/internationalization)) | CLDR 30.0.3 | Unicode 9.0 |
+| Android 7.0 - 7.1 (API levels 24 - 25) | ICU4j 56 ([ref](https://developer.android.com/guide/topics/resources/internationalization))| CLDR 28 | Unicode 8.0 |
+
+<br />
+
+**Pre-24 platforms**

```

Comment:
```suggestion
### Pre-24 platforms
```

🤔   I haven't tried it locally but I imagined you can gain some proper margin by making it a header? (Same applies to the first table head)